private/model/api/smoke.go

```diff
@@ -59,10 +59,12 @@ func (a *API) APISmokeTestsGoCode() string {
 	a.AddImport("context")
 	a.AddImport("testing")
 	a.AddImport("time")
+
 	a.AddSDKImport("aws")
 	a.AddSDKImport("aws/awserr")
 	a.AddSDKImport("internal/awstesting/integration")
-	a.AddImport(a.ImportPath())
+	a.AddSDKImport("service", a.PackageName())
+	a.AddSDKImport("service", a.PackageName(), TypesPkgName)
```

Comment:
I think changing the `a.AddImport(a.ImportPath())`, to `a.AddImport(a.ImportPath())` will prevent providing a base import path that doesn't start with `service`.